### ISSUE TITLE
Entering CRM customers requests

### ISSUE THREAD

**Issue by inverted-capital** [dreamcatcher-tech/dreamcatcher-tech.github.io#59]
Our CRM customer wants to move their work requirements into the Dreamcatcher.

I have placed a very rough draft into [R23](https://dreamcatcher-tech.github.io/dust/Requests/R23) and need to figure out how to handle this and the rest of their requests.  These requests will come with timelines and funding.  We need to be clear about the privacy information our customer is comfortable divulging.

Further to our phone calls about how to get their information and payments moving through the system, should be make a private fork with their information in it, then get them to ok it before releasing to the public ?

**Comment by TemperBead** [comment]
Lots to unpack here.

Initial thoughts on this but will probably need a call.

There's a line to be tread here.  

One of our main tenents is that holding information private is often the first impulse for a company because they don't want to divulge info to competitors, but we're arguing that they should make it as public as possible so they get the greatest network effect, and even beyond that in allowing reuse through attribution.

It would be great if we can test how palitable that is to a real company, and what arguments we could put forth to convince them.

Also, realistically, who's looking at us now?  The chance that a competitor will get useful information from Requests is vanishingly small.  

Finally, we don't have a network, so it doesn't really matter as no one else will be picking up the work.  We could keep it all secret.  But if we do that we're losing out on an opportunity to see how strong this privacy instinct is.

If I were in their position I think I'd be comfortable with the Requests being made public after they were met, knowing that we're hand-picking who's doing the work and that there isn't a marketplace.  This is your fork/check/merge proposal.  However, if everyone followed that route, we'd miss out on some of the major strengths of the DC.

A reasonable middle ground and fair test of the stickiness of corporate info-hoarding though would be to suggest that we put out the Requests publicly, but obfuscate their identity.  It may be that it would be hard to argue in terms of benefit to them, but it would be a test for us.

Obviously, eventually, we want to leverage permissionless/transparent, tracked by Attribution, but this is the first time we've had a chance to test whether that core assumption in terms of Buyer-side.  Again, we don't have a network, so it's not a deal breaker.  But think we should propose and champion the concept and see where they have real problems with it.

**Comment by inverted-capital** [comment]
I agree with your 4 steps to deal with this instance.  I think long term we need to build in differential privacy, with layers set along the lines of what you said:
1. Full view
2. names and amounts removed
3. code / solutions removed

Something like that.  Then you can ratchet up as the game unfolds.

**Comment by TemperBead** [comment]
Cool.  Will be very interested to understand more about the urge to privacy.  I've had that myself in the past, but on consideration it just seems, well, a bit daft...  

This is your ticket, so by etiquette will leave it for you to close or keep open if there's more to go into here.
